### DISCUSSION TITLE
What tools is everyone using for writing fennel?

### DISCUSSION THREAD

**Discussion by varunaswal** [agzam/spacehammer#177] [Q&A]
I've been making some additions to my config and have yet to find good tools that support fennel in either VSCode or Emacs. The code in the repo appears to have consistent formatting, how is this achieved? I'm new to Fennel/Lisp and feel like it would be a lot easier to use with autoformatting. 

Thanks for the project btw! It's slowly becoming my primary interface with my system 😌

**Comment by Grazfather** [comment]
I just use neovim with the fennel plugs, and just use `=` to format (manually). Formatting definitely isn't 100% consistent.

We have a repl feature, it can make it a lot more fun to develop, though admittedly I don't use it often. With autoload I find just saving is often good enough.
